lexicons/app/bsky/feed/postgate.json

```diff
@@ -16,16 +16,16 @@
             "format": "at-uri",
             "description": "Reference (AT-URI) to the post record."
           },
-          "detachedQuotes": {
+          "detachedEmbeddingUris": {
             "type": "array",
             "maxLength": 50,
             "items": {
               "type": "string",
               "format": "at-uri"
             },
-            "description": "List of detached quote post URIs."
+            "description": "List of AT-URIs embedding this post that the author has detached from."
           },
-          "quotepostRules": {
+          "embeddingRules": {
```

Comment:
On threadgate records the corresponding property is called `allow`, which contains the rules determining whether a given reply is allowed.

I am not opposed to `embeddingRules`, though if we want to be really consistent with threadgates it might become something like `allowEmbedding`.  I'll leave it up to you and @brianolson whether you want to adopt that change.

Reply:
a few people noted that an empty list of allow rules being implicitly deny is less clear than an explicit deny rule. it's also a little weird because nil-record is different than empty-record. so I'm in favor of the explicit deny rule. I _think_ this implicitly makes the logic that you can have deny rules _and_ allow rules, and I'd interpret an allow rule as superseding the deny rule. `{deny *, allow foo, allow bar}` _does_ allow foo and bar